Created facet menu for company names!

liangdanica opened the pull request:
I mainly just created another facet menu for company names, but here is some other stuff that has been changed:
- I also added a range element index in the test/ml.config.xml file for company, so remember to run "ml local bootstrap"
- Calendar pop-up was commented out before, but I uncommented it and added some more stuff to it which also makes the calendar icon clickable to view the pop-up!

graceyin1218 commented on src/index.html:
```diff
@@ -121,7 +121,7 @@ <h4 class="panel-title"><a data-toggle="collapse" href="#collapse1">Select Indus
           </div>
```
Indents from here on seem sort of off...

graceyin1218 commented on src/public/map.js:
```diff
@@ -356,21 +386,21 @@ function formatPopup(properties) {
     str += "<br>";
-  } else if (properties.features && properties.features.length === 0) {
+    }
+   else if (properties.features && properties.features.length === 0) {
```
Feel free to ignore me - this probably makes very little difference, but line 390 is off by one space.... 😇

graceyin1218 commented on src/public/map.js:
```diff
@@ -380,17 +410,29 @@ $(function filterDate() {
+
+  $('input[name="datefilter"]').on('apply.daterangepicker', function (ev, picker) {
+
+      $(this).val(picker.startDate.format('MM/DD/YYYY') + ' - ' + picker.endDate.format('MM/DD/YYYY'));
```
Feel free to ignore me again 😅 

Line 422 is over-indented, as are certain lines in the next few blocks of code..

liangdanica commented:
Okay! I fixed the indentation. Hopefully better?

liangdanica commented:
Wait just kidding, still fixing......

graceyin1218 commented:
😅 😅 😅

graceyin1218 commented on src/index.html:
```diff
@@ -140,15 +139,31 @@ <h4 class="panel-title"><a data-toggle="collapse" href="#collapse2">Select Featu
             </div>
```
Wait, what divs are these closing?

graceyin1218 commented:
Uh... Darn... @liangdanica? Stuff is breaking when I try ml local boostrap...

ERROR: 500 "Internal Server Error"
ERROR: {"errorResponse":{"statusCode":500, "status":"Internal Server Error", "messageCode":"XDMP-UNEXPECTED", "message":"XDMP-UNEXPECTED: (err:XPST0003) Unexpected token syntax error, unexpected DoubleDash_, expecting XMLCommentEnd_", "messageDetail":{"messageTitle":"Unexpected token"}}}



Also, I just noticed this: in map.js, map.on("zoomend", ...) and map.on("drag" ,...), which update the markers on the map, have been deleted........❓ ❔ ❓

liangdanica commented:
What does your ml.config.xml look like? I kept getting an Internal Server Error too until I added 'company' as a range element index and ran ml local bootstrap. Wait so it gives you that when you run ml local bootstrap?

Yeah, I added that as an issue -- to delete the zoomend and drag because Joe didn't like that every time we panned out or dragged elsewhere on the map, that it would reset. We can add it back in if you like for now, but I thought we were planning on deleting it and adding a general 'reset' button instead.

kmcostel commented:
Hey Grace, I don't get that error so we can look once you get here.... I deleted those map events because it'll be simpler to not redraw the map for those events and instead just search the entire map if no drawn shapes are present because speed isn't an issue. Pretty much every other event on the map updates the markers.

graceyin1218 commented:
Found it -> one comment in ml-config.xml isn't being interpreted as a comment on my laptop... (the --- was messing things up.)

Maybe it's a mac vs windows thing? eh.. 

I'll fix it.
